src/graphics/reproject.js

```diff
@@ -0,0 +1,84 @@
+import { Vec4 } from '../math/vec4.js';
+
+import {
+    TEXTURETYPE_RGBM, TEXTURETYPE_RGBE,
+    PIXELFORMAT_RGB16F, PIXELFORMAT_RGB32F, PIXELFORMAT_RGBA16F, PIXELFORMAT_RGBA32F
+} from './graphics.js';
+import { drawQuadWithShader } from './simple-post-effect.js';
+import { shaderChunks } from './chunks.js';
+import { RenderTarget } from './render-target.js';
+
+// get a coding string for texture based on its type and pixel format.
+var getCoding = function (texture) {
+    switch (texture.type) {
+        case TEXTURETYPE_RGBM:
+            return "RGBM";
+        case TEXTURETYPE_RGBE:
+            return "RGBE";
+        default:
+            switch (texture.format) {
+                case PIXELFORMAT_RGB16F:
+                case PIXELFORMAT_RGB32F:
+                case PIXELFORMAT_RGBA16F:
+                case PIXELFORMAT_RGBA32F:
+                    return "Linear";
+                default:
+                    return "Gamma";
+            }
+    }
+};
+
+/**
+ * @static
+ * @function
+ * @name pc.reproject
```

Comment:
I'm wondering if this name is clear enough. I mean, reproject what? 😄

Reply:
Do you have any suggestions? reprojectEnv?

Reply:
`reprojectTexture`? I mean, it takes textures as arguments.

Reply:
And it'd go with the pre-existing [`drawTexture`](https://developer.playcanvas.com/en/api/pc.html#drawTexture).

Reply:
sold! :)